### PR TITLE
mini project and hackerrank exercises - java oop

### PR DESCRIPTION
**Pull request by mike-angek96**
HackerRank exercises  and Mini project using in inheritance (Superclass and Subclass).  Java OOP

### CODE REVIEW

File: MiguelAngelVidalLiborio/Dia4-26Agosto/HackerRank/JavaInheritanceFirst.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package hackerrank;
+
+class Animal {
```
Comment (edbermeo) [inline MiguelAngelVidalLiborio/Dia4-26Agosto/HackerRank/JavaInheritanceFirst.java:3]: Use a file per class

File: MiguelAngelVidalLiborio/Dia4-26Agosto/HackerRank/JavaInheritanceSecond.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package hackerrank;
+
+class Arithmetic {
```
Comment (edbermeo) [inline MiguelAngelVidalLiborio/Dia4-26Agosto/HackerRank/JavaInheritanceSecond.java:3]: Use a file per class

File: MiguelAngelVidalLiborio/Dia4-26Agosto/HackerRank/JavaOutPutFormatting.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+public class JavaOutPutFormatting {
+
+	public static final Scanner scanner = new Scanner(System.in);
+
+	public static void main(String[] args) {
+		int limitLoop=3;
+		
+		System.out.println("please enter your programming language and how much you love it");
+		System.out.println("================================");
```
Comment (edbermeo) [inline MiguelAngelVidalLiborio/Dia4-26Agosto/HackerRank/JavaOutPutFormatting.java:13]: Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings

File: MiguelAngelVidalLiborio/Dia4-26Agosto/MiniProject/javaoop/Main.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package javaoop;
+
+public class Main {
+
+	public static void main(String[] args) {
+		Course myCourse = new Course();
+		myCourse.welcome();
+		myCourse.welcome("heey whats up");
+
+		System.out.println("");
```
Comment (edbermeo) [inline MiguelAngelVidalLiborio/Dia4-26Agosto/MiniProject/javaoop/Main.java:10]: What is the functionality of this?

**Review by edbermeo — CHANGES_REQUESTED** [review]

**Review by luisalanmz — APPROVED** [review]
LGTM . Just answer the comments

**Review by edbermeo — APPROVED** [review]

**Pull request merged**